docs/src/pages/Options.mdx

```diff
@@ -1050,6 +1050,45 @@ Automatically call **[disableInput()](docs/api/Flicking#disableInput)** on initi
   </ColumnItem>
 </Columns>
 
+### changeOnHold
+Change active panel index on mouse/touch hold while animating.
```

Comment:
Is it only necessary for hold? It seems like a good feature to change the activePanel even when not on hold.

Reply:
Umm... like when? Can you show me some example cases?